### PR TITLE
Get dynaconf list into foreman-debug

### PR DESCRIPTION
**Pull request by sjha4**

### CODE REVIEW

**Comment by parthaa** [comment]
Works well. Tested on my dev box, APJ

**Comment by ekohl** [comment]
foreman-debug is sort of deprecated with the hope to [sosreport](https://github.com/sosreport/sos). That doesn't block this, but please work with them for native Pulpcore support in the tool.

**Comment by sjha4** [comment]
@ekohl: Thanks for the direction. I have opened https://github.com/sosreport/sos/issues/2266 to track this.

**Comment by jlsherrill** [comment]
Merging!

File: packages/katello/katello/katello-debug.sh [hunk 1]
```diff
@@ -73,6 +73,7 @@ add_files /etc/default/pulp*
 add_files /etc/pulp/settings.py
 add_cmd "systemctl list-units -t service --full  | grep pulpcore" "pulpcore-services"
 add_cmd "systemctl cat pulpcore*" "pulpcore_service_files"
+add_cmd "sudo -u pulp PULP_SETTINGS='/etc/pulp/settings.py' DJANGO_SETTINGS_MODULE='pulpcore.app.settings' dynaconf list" "dynaconf_list"
```
Comment (ekohl) [inline packages/katello/katello/katello-debug.sh:76]: I do wonder what value this is if `/etc/pulp/settings.py` is already present.

**Pull request merged**